DuckDuckGo/Fire/View/FirePopoverViewController.swift

```diff
@@ -88,7 +88,7 @@ final class FirePopoverViewController: NSViewController {
                                                          historyCoordinating: historyCoordinating,
                                                          fireproofDomains: fireproofDomains,
                                                          faviconManagement: faviconManagement,
-                                                         tld: ContentBlocking.shared.tld)
+                                                         tld: ContentBlocking.shared.tld, contextualOnboardingStateMaching: Application.appDelegate.onboardingStateMachine)
```

Comment:
nip: probably move `contextualOnboardingStateMaching:` to next line to keep consistency in the format